components/org.wso2.ppaas.manager.console/console/controllers/menu/menu_generator.jag

```diff
@@ -100,6 +113,14 @@ function metroGenerator(context, jsonobject, permissions) {
                     hasPermission = intersection(jsonPermission.permissionPaths, permissions);
 
             if (hasPermission.length > 0 && jsonPermission.context == context) {
+                if (jsonPermission.title == "Monitoring") {
```

Comment:
Notice in some other places also checking the condition against the "Monitoring". Better if you make this as  a constants variable.